lib/src/bt_device.dart

```diff
@@ -303,6 +303,23 @@ class GYWBtDevice with ChangeNotifier implements Comparable<GYWBtDevice> {
     await _sendBTCommand(command);
   }
 
+  Future<void> clearScreen({Color? color}) {
```

Comment:
You must describe the method for the dartdoc library generation
```suggestion
  /// Reset the content of the screen and its background color
  Future<void> clearScreen({Color? color}) {
```

Reply:
Can we add a CI action for that?